shuffle/move/examples/e2e/message.test.ts

```diff
@@ -59,15 +69,20 @@ Deno.test("Ability to set NFTs", async () => {
     secondUserContext.address,
     creator,
     creationNum,
+    "TestNFT",
+    context.defaultUserContext,
+    context.defaultUserContext.address,
   );
-  transferTxn = await devapi.waitForTransactionCompletion(transferTxn.hash);
+  transferTxn = await devapi.waitForTransaction(transferTxn.hash);
   assert(transferTxn.success);
 
   // Check receiver has the nft
   const receiverNFTs = await main.decodedNFTs(secondUserContext.address);
   assertEquals(receiverNFTs[0].content_uri, contentUri);
   // Check sender nft_collection is empty
-  const senderNFTs = await main.decodedNFTs();
+  const senderNFTs = await main.decodedNFTs(
+    context.defaultUserContext.address,
```

Comment:
Why are we passing in the default that it gets auto set to anyways?

Reply:
Just so that the code is more explicit / for tutorial purposes